roles/dhcp-server/templates/boxes.conf.j2

```diff
@@ -0,0 +1,20 @@
+# /etc/dnsmasq.d/boxes.conf
+#
```

Comment:
These comments seem pointless. The one below too. Replace with `# {{ ansible_managed }}` ?

Reply:
removed the ones below.

I like to have the location of a file in the file like that.
mostly for when I share them with someone else we know where it came from and where it goes, both for debugging (why isn't this setting respected?)  and showing someone else my setup (here is how I pxe boot the installer...)